src/Products/dataReducer.js

```diff
@@ -0,0 +1,46 @@
+export const dataReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_DATA":
+      return { ...state, data: action.payload };
+    case "ADD_CATEGORIES":
+      return { ...state, categories: action.payload };
+    case "STATUS":
+      return {
+        ...state,
+        status: action.payload,
+      };
+    case "CLEAR_FILTERS":
+      return {
+        ...state,
+        showInventoryAll: true,
+        showFastDelivery: false,
+        sortBy: null,
+        priceRange: 1000,
+        level: "",
+        ratings: 5,
+        searchString: "",
+        category: "",
```

Comment:
I have a suggestion, make filters an object to make it scalable.

Reply:
> Hi Prerana,
> I have reviewed your PR. Great work! I have few common suggestions listed here:
> 
> 1. You can keep all the components inside the Component folder so that the src folder looks clean. Add index.js file and re-export all the components from there, so that App.js imports will look cleaner
> 2. Always commit files to main which does not require review, like logo images, index.html. So that they don't show in changed files.
> 3. Create constants for fixed values like API_URL, so that in the future if you want to update any value, it becomes easy to change at one place
> 4. I have a suggestion always use rem instead of px unit in css
> 5. We should not write handler in function name when the name is self explanatory e.g. signupUserHandler -- SignupUser

Hey ankita, thanks a lot for reviewing. All your comments were super helpful to me! Thanks again